app/controllers/comments_controller.rb

```diff
@@ -13,7 +13,11 @@ def index
   def show; end
 
   def new
-    @comment = @idea.comments.build
+    if @idea.approved?
```

Comment:
It might be better to make this a guard clause to avoid a lengthy if..else like we have in the `create` method?

```
redirect_to(@idea, notice: 'Comments can only be added to approved ideas.') unless @idea.approved?
return if performed?

@comment = @idea.comments.build
```

This is probably more applicable in the `create` method, but applies here too.

Reply:
Done 👍